<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialParam</class>
 <widget class="QWidget" name="InitialParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="flag_group">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Чем ходить</string>
   </property>
   <widget class="QRadioButton" name="flag_crest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Нули</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="flag_zeros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Плюсы</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="firststep_group">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>59</y>
     <width>120</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Первый ход</string>
   </property>
   <widget class="QRadioButton" name="first_hum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Человек</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="first_comp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Комп</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QSlider" name="hadr_slider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сложность</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
